### ISSUE TITLE
Decide what to do with master and development code

### ISSUE THREAD

**Issue by philmmanjaro** [manjaro/manjaro-tools#276]
We should review both branches and decide what to do about them.

**Comment by oberon2007** [comment]
I think it makes little sense to move to dev branch. It is very different and when I last tested there were some issues afai remember.
diff is huge and practically impossible to compare ...
master seems to be working fine. I would opt to stick with it and go from there.

**Comment by philmmanjaro** [comment]
**Master** has to be reviewed also. We will see when we find time. I've released **v0.15.4** for now.

**Comment by oberon2007** [comment]
ah well I meant master 😉

**Comment by oberon2007** [comment]
I don't see why we need to move everything again to different files, change var names. It's like starting over again. The diff is a total mess. No idea why we would need to do that.

**Comment by philmmanjaro** [comment]
Maybe some good code is there. Else we ditch it and go with **v0.15.4** ;)

**Comment by Chrysostomus** [comment]
Just read through the diff. Much moving code around. Help command gives info about -U flag. Cleaning out the parts that configure thus. Some work towards supporting calamares netinstall better. 

If master works as is and devel does not, I would stick with master and ditch devel. Maybe incorporate some specific commits. But in principle take master and work forward from there. 

Btw, is netinstall with calamares something that we are actively working towards, or was that more of artoos own thing?

**Comment by philmmanjaro** [comment]
I think the **net-install** got modified upstream already, so we have to check if it still works. Anyway, if we find some useful code, we may use it. I had no time yet to even review the **master** changes yet.

**Comment by oberon2007** [comment]
my understanding is that we were considering to go on from stable-15.x branch - which is again a LOT different from master! 😉

**Comment by philmmanjaro** [comment]
Yep. That was the old code. Master was some like **v0.16.0** ish, which might need some backports from **v0.15** series. If no need we may ditch both and stick with **v0.15**.
